Dockerfile

```diff
@@ -0,0 +1,65 @@
+# syntax = docker/dockerfile:1
```

Comment:
i'm seeing some differences with the one generated with a default rails app

any reason to use the `dockerfile-rails ` one instead of the default one?

here is the diff:

```diff
diff --git a/Dockerfile b/mi_carrera_new/Dockerfile
index 6a6ef5c..a40ad90 100644
--- a/Dockerfile
+++ b/mi_carrera_new/Dockerfile
@@ -2,19 +2,16 @@
 
 # Make sure RUBY_VERSION matches the Ruby version in .ruby-version and Gemfile
 ARG RUBY_VERSION=3.2.2
-FROM ruby:$RUBY_VERSION-slim as base
+FROM registry.docker.com/library/ruby:$RUBY_VERSION-slim as base
 
 # Rails app lives here
 WORKDIR /rails
 
 # Set production environment
 ENV RAILS_ENV="production" \
-    BUNDLE_WITHOUT="development:test" \
-    BUNDLE_DEPLOYMENT="1"
-
-# Update gems and bundler
-RUN gem update --system --no-document && \
-    gem install -N bundler
+    BUNDLE_DEPLOYMENT="1" \
+    BUNDLE_PATH="/usr/local/bundle" \
+    BUNDLE_WITHOUT="development"
 
 
 # Throw-away build stage to reduce size of final image
@@ -22,16 +19,16 @@ FROM base as build
 
 # Install packages needed to build gems
 RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y build-essential libpq-dev
+    apt-get install --no-install-recommends -y build-essential git libpq-dev pkg-config
 
 # Install application gems
-COPY --link Gemfile Gemfile.lock ./
+COPY Gemfile Gemfile.lock ./
 RUN bundle install && \
-    bundle exec bootsnap precompile --gemfile && \
-    rm -rf ~/.bundle/ $BUNDLE_PATH/ruby/*/cache $BUNDLE_PATH/ruby/*/bundler/gems/*/.git
+    rm -rf ~/.bundle/ "${BUNDLE_PATH}"/ruby/*/cache "${BUNDLE_PATH}"/ruby/*/bundler/gems/*/.git && \
+    bundle exec bootsnap precompile --gemfile
 
 # Copy application code
-COPY --link . .
+COPY . .
 
 # Precompile bootsnap code for faster boot times
 RUN bundle exec bootsnap precompile app/ lib/
```

Reply:
I'm ok either way.

I think if i were creating a new app i would just use the default rails one, and just make tweaks if/when we need them. But if you think the `dockerfile-rails` one is better, let's go with it

Reply:
I agree with that! I'd keep an eye on `dockerfile-rails` though 👀

Reply:
Updated! https://github.com/cedarcode/mi_carrera/pull/337/commits/a30a065478f85fbc67cd636131e315f762e75f90